.github/workflows/docs.yml

```diff
@@ -68,7 +68,16 @@ jobs:
         sudo apt update
         sudo apt install -y doxygen
 
+        # Install fastffi for Java sdk compilation
+        cd /tmp/
+        git clone https://github.com/alibaba/fastFFI.git
+        cd fastFFI
+        git checkout a166c6287f2efb938c27fb01b3d499932d484f9c
+        export PATH=${PATH}:${LLVM11_HOME}/bin
+        mvn clean install -DskipTests -Dmaven.antrun.skip=true
```

Comment:
Use `mvn --quiet`, otherwise there would be tons of logs (not every useful) in the CI.

Please check all usage of `mvn` command in your previous commit as well.